File: Codigo Java/EjemploGato/Gato.java
package EjemploGato;

public class Gato {
    String name;
    String sex;
    int age;
    int weight;
    String color;
    String texture;

    Gato(String name, String sex, int age, int weight, String color, String texture){
        this.name=name;
        this.sex=sex;
        this.age=age;
        this.weight=weight;
        this.color=color;
        this.texture=texture;
    }
    void eat(){
        System.out.println(this.name + " esta comiendo");
    }
}

File: python/EjemploGato/Gato.py
class Gato:
    def __init__(self, name, sex, age, weight, color, texture):
        self.name = name
        self.sex = sex
        self.age = age
        self.weight = weight
        self.color = color
        self.texture = texture

        mi_gato = Gato(name="Ariana", sex="Hembra", age=7, weight=4.5, color="calico", texture="Suave")

        print(mi_gato.name)
